Utilities/Maintenance/UpdateRemoteModules.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+program_name=$0
+
+function usage {
+  echo "Usage: ${program_name} "
+  echo "  param1 Remote modules' path"
+  exit 1
+}
+
+# Display usage
+if [ ${#@} != 1 ]; then
+  usage
+fi
+
+remote_modules_path=$1
```

Comment:
We could use `BASH_SOURCE` and the relative path of *ITK/Modules/Remote* from this script to avoid the need for this argument -- it will always be the same relative directory. See e.g.:

https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/GitSetup/setup-upstream#L32

Reply:
Done in b68921c.